### ISSUE TITLE
Possible goroutine leaking

### ISSUE THREAD

**Issue by FrontMage** [gernest/wow#14]
It's OK using on a one time command, while using wow with prompt, it consumes more cpu usage every time it starts, doesn'st seem to clean up it's goroutines.

**Comment by falzm** [comment]
Agreed, there is definitely something wrong with the goroutines created in the `Start()` method. Here is pprof trace of about 2 minutes running *wow*:

<img width="597" alt="screenshot 2019-01-07 at 14 38 47" src="https://user-images.githubusercontent.com/1122379/50771194-2f4ef000-128a-11e9-8e88-e88b7a003328.png">
<img width="1666" alt="screenshot 2019-01-07 at 14 39 24" src="https://user-images.githubusercontent.com/1122379/50771193-2f4ef000-128a-11e9-9973-e75e9818b075.png">
